### DISCUSSION TITLE
Unable to send traces to otel collector from application in docker container

### DISCUSSION THREAD

**Discussion by stephenhong** [open-telemetry/opentelemetry-collector-contrib#5068] [Q&A] [answered]
Hello team, I have a node.js application with @opentelemetry/exporter-collector to send traces to my otel collector. It works fine and traces are going to the collector. However, when I put that node.js application in a docker container and run it, it doesn't send traces to my collector. Do applications in containers not work with otel collectors or am I missing some configuration to make this work?

**Comment by stephenhong** [comment, marked as answer]
I was able to resolve this issue

**Reply by vesta-edgar-garcia** [reply]
how?
